app/conllu_to_json_converter.py

```diff
@@ -65,6 +72,10 @@ def __read_conllx(input_path, use_morphology=False, n=0):
                     # Because of error i Danish format?
                     tag = pos
```

Comment:
Is it overwriting `tag` on line 70? If it's right, do we need of line 70 (`tag = tag+'__'+morph if use_morphology else tag`)?